testutils/factories/appointment.ts

```diff
@@ -24,10 +25,15 @@ class AppointmentFactory extends Factory<Appointment> {
   }
 }
 
+const defaultAppointmentDeliveryType: AppointmentDeliveryType = 'VIDEO_CALL'
+
 export default AppointmentFactory.define(({ sequence }) => ({
   id: sequence.toString(),
   appointmentTime: new Date().toISOString(),
   durationInMinutes: 60,
+  // For some reason the compiler complains if I write 'VIDEO_CALL' inline
+  appointmentDeliveryType: defaultAppointmentDeliveryType,
```

Comment:
I think it assumes that 'VIDEO_CALL' is a string rather than a AppointmentDeliveryType, because the type is defined as `any` in the factory. I think your solution is the only way around it.